### DISCUSSION TITLE
Cannot write command into connection after manual backup

### DISCUSSION THREAD

**Discussion by DharaniSanjai** [redisson/redisson#4995] [Q&A]
I am getting the following error after I manually removed the Redis cluster and restored it from backup. Currently a restart of the application is required after the manual backup. May I know where is the issue ? Also how can I refresh the connection/configuration? 

Unable to write command into 
connection! Node source: NodeSource [slot=xxxx, addr=null, 
redisClient=null, redirect=null, entry=null], connection: 
RedisConnection@xxxxx
[redisClient=[addr=rediss://xxxxx:6379],
 channel=[id: xxxxx, L:0.0.0.0/0.0.0.0:xxxx], currentCommand=null], command: (GET), params: [types_version] after 10 retry attempts"

The client configuration is as follows

Config config = new Config();
    config.setNettyThreads(64)
        .useClusterServers()
        .setScanInterval(2000)
        .setRetryAttempts(10)
        .setReadMode(ReadMode.MASTER)
        .addNodeAddress(redisNodes);
    config.setCodec(new StringCodec());
    return Redisson.createReactive(config);

**Comment by mrniko** [comment]
Can you share the logs with `trace` logging level for `org.redisson` package?
